app/callbooker/_process.py

```diff
@@ -94,14 +94,15 @@ async def get_or_create_deal(company: Company, contact: Contact) -> Deal:
     config = await get_config()
     if not deal:
         match company.price_plan:
-            case Company.PP_PAYG:
+            case plan if Company.PP_PAYG in plan:
```

Comment:
If `price_plan` is being validated from the tc2 webhook then better to use pydantic validator to remove the `-`